src/cdk/collections/selection.ts

```diff
@@ -39,7 +39,8 @@ export class SelectionModel<T> {
   constructor(
     private _multiple = false,
     initiallySelectedValues?: T[],
-    private _emitChanges = true) {
+    private _emitChanges = true,
+    private _trackBy?: (model) => any) {
```

Comment:
Looks like there aren't any unit tests that cover this new `trackBy` behavior; this would be necessary since it's part of the main library rather than cdk-experimental